lib/utils.js

```diff
@@ -34,5 +40,72 @@ module.exports = {
    * Get the default line height (font size * 1.2) for font styles
    * missing a user-defined line height.
    */
-  calculateDefaultLineHeight: (fontAttrs) => parseFloat((fontAttrs.size * 1.2).toFixed(2))
+  calculateDefaultLineHeight: fontAttrs => parseFloat((fontAttrs.size * 1.2).toFixed(2)),
+
+  /**
+   * Maps keys of a color object from r,g,b,a
+   * to green, blue, red and alpha
```

Comment:
Why would we need to map colors to longform?
Output should always be rgb in shortform

Reply:
lol, other way around

Reply:
The reasoning behind it is that in this manner I can make use of the formatColor function which expects the full names. Should i rewrite that part?

Reply:
I would get longform -> shortform. Would not know why you need shortform -> longform.